tomcat85x/src/main/groovy/com/bmuschko/gradle/tomcat/embedded/Tomcat85xServer.groovy

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2011 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.bmuschko.gradle.tomcat.embedded
+
+import java.lang.reflect.Constructor
+
+/**
+ * Tomcat 85x server implementation.
+ *
+ * @author Benjamin Muschko
+ * @author Andrey Bloschetsov
+ * @author Martin Flower
+ */
+class Tomcat85xServer extends BaseTomcat7xPlusImpl {
```

Comment:
I don't see any difference to `Tomcat8xServer` except the version. Can't we just extend from that class and only override `getVersion()`?